web-marketplace/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -1,3 +1,4 @@
+import { useMemo } from 'react';
```

Comment:
It looks like this import isn't being used.

Reply:
removed